### PR TITLE
Tangany community story

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+**Jens:** Integrating the fast-moving world of digital asset custody into the more slow-paced world of traditional finance.
+
+**Dan:** Tell me about your role at Tangany.
+
+**Jens:** I have been a software developer at Tangany for over four years now (wow, time flies...) and have been part of a team building our Partner Portal where customers can manage their access to the APIs provided by Tangany.
+
+**Dan:** How do you use FusionAuth? OAuth? User management? Social sign-on? Something else?
+
+**Jens:** We use FusionAuth for our user management and OAuth 2.0 flows to authenticate our partners in the B2B portal.
```
Comment (bradmccarty) [inline astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:44]:
```suggestion
**Jens:** We use FusionAuth for our user management and [OAuth 2.0](/articles/oauth/modern-guide-to-oauth) flows to authenticate our partners in the B2B portal.
```

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+**Jens:** We were using another authentication provider.
+
+**Dan:** How are users added to FusionAuth? Is it self-service registration or are you using the APIs to create and register users?
+
+**Jens:** We use the API to add new users to our system.
+
+**Dan:** What security mechanisms or features are on your near-term roadmap?
+
+**Jens:** Step up authentication will enable us to provide a more secure workflow within our applications. We tested this FusionAuth feature during the vendor selection phase.
```
Comment (bradmccarty) [inline astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:62]:
```suggestion
**Jens:** [Step up authentication](/docs/lifecycle/authenticate-users/multi-factor-authentication) will enable us to provide a more secure workflow within our applications. We tested this FusionAuth feature during the vendor selection phase.
```

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+> Due to regulatory requirements, we required a provider capable of self-hosting.
+
+**Dan:** Why did you choose FusionAuth over the alternatives?
+
+**Jens:** FusionAuth had a more stable developer experience and seemed to be a good balance between reliability, effort to integrate, feature set, and price. Due to regulatory requirements, we required a provider capable of self-hosting.
```
Comment (bradmccarty) [inline astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:68]:
```suggestion
**Jens:** FusionAuth had a more stable developer experience and seemed to be a good balance between reliability, effort to integrate, feature set, and price. Due to regulatory requirements, we required a provider capable of [self-hosting](/blog/when-to-self-host).
```

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
```
Comment (bradmccarty) [inline astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:70]: Should we remove this question and the response?

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+Unfortunately, we continue to encounter challenges in monitoring audit and login logs. We had to develop custom solutions to store them within logging platforms due to our self-hosted environment.
+
+**Dan:** How do you run FusionAuth (Kubernetes, standalone server, behind a proxy, etc)?
+
+**Jens:** For regulatory compliance, we've opted to operate FusionAuth within a self-hosted cloud environment.
+
+**Dan:** In that cloud environment, are you using the docker image, an RPM, or something else?
+
+**Jens:** We are using Terraform and the FusionAuth Docker images for deployments.
```
Comment (bradmccarty) [inline astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:82]:
```suggestion
**Jens:** We are using [Terraform](/docs/operate/deploy/terraform) and the FusionAuth Docker images for deployments.
```

**Review by bradmccarty — CHANGES_REQUESTED** [review]
Some internal links, and one question. Otherwise LGTM.

File: astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx (hunk 4 above)
Comment (mooreds) [reply astro/src/content/blog/tangany-fusionauth-self-hosted-regulatory.mdx:70]: Hi @bradmccarty, I think we should leave it in. I feel like stories are always better if they have real world challenges outlined. They also filed an issue, which I linked to, which highlights our input process.

**Review by bradmccarty — APPROVED** [review]

**Pull request merged**